frontend/app/src/pages/superadmin/tableComponent/index.tsx

```diff
@@ -258,8 +257,13 @@ export const MyTable = ({
             <img src={copygray} alt="" width="16.508px" height="20px" />
             <LeadingTitle>
               {' '}
-              {bountiesLength}{' '}
-              <AlternativeTitle> {bountiesLength === 1 ? 'Bounty' : 'Bounties'}</AlternativeTitle>{' '}
+              {bounties.length}{' '}
```

Comment:
I'm noticing these weird extra spaces `{' '}` ik it wasn't added in this PR but we should probably remove them

Reply:
So you're suggesting I remove them? Is that is it then okay no issue

Reply:
Done